src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-otlp-troubleshooting.mdx

```diff
@@ -8,6 +8,8 @@ tags:
   - Troubleshoot
 metaDescription: Troubleshoot common OTLP ingest errors
 freshnessValidatedDate: 2024-05-06
+redirects:
+ - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-troubleshooting/
```

Comment:
@jack-berg There aren't any docs references for the deleted troubleshooting doc, but in case of other external references (i.e., the UI), it'll at least get redirected to the OTLP troubleshooting doc.

Reply:
Great!